frameworks/helloworld/tests/test_sanity.py

```diff
@@ -32,12 +32,20 @@ def configure_package(configure_security):
     try:
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
 
-        sdk_upgrade.test_upgrade(
-            PACKAGE_NAME,
-            PACKAGE_NAME,
-            DEFAULT_TASK_COUNT,
-            service_name=FOLDERED_SERVICE_NAME,
-            additional_options={"service": {"name": FOLDERED_SERVICE_NAME}})
+        if shakedown.dcos_version_less_than("1.9"):
+            # Hello-world before 1.9 isn't supported.
```

Comment:
Basically yes, as we should be testing it (regardless of it being able to pass without us testing it)